src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -49,25 +51,71 @@ type Props = {
     tagged_message: string,
     translatedTaggedMessage?: string,
     translations?: Translations,
+    type?: typeof COMMENT_TYPE_DEFAULT | typeof COMMENT_TYPE_TASK,
     userHeadlineRenderer?: React.Node => React.Element<typeof FormattedMessage>,
 };
 
 type State = {
+    isConfirming?: boolean,
     isEditing?: boolean,
-    isFocused?: boolean,
     isInputOpen?: boolean,
 };
 
 class Comment extends React.Component<Props, State> {
+    static defaultProps = {
+        type: COMMENT_TYPE_DEFAULT,
+    };
+
     state = {
+        isConfirming: false,
         isEditing: false,
-        isFocused: false,
         isInputOpen: false,
     };
 
+    onDeleteConfirm = (): void => {
+        const { id, onDelete, permissions } = this.props;
+
+        if (onDelete) {
+            onDelete({ id, permissions });
+        }
+    };
+
+    onDeleteCancel = (): void => {
```

Comment:
Some methods are using `on` and some are using `handle`. Can we standardize on the latter?